config/config.go

```diff
@@ -10,4 +10,41 @@ type Config struct {
 	LibraryPattern      *regexp.Regexp
 	ExcludeFromGrouping *regexp.Regexp
 	SmapMapper          sourcemap.Mapper
+	Agents              strContainer
```

Comment:
we've used a `set.Set` from github.com/fatih/set in other places. Maybe that fits here too?

Reply:
We've only used the `set.Set` in tests so far. As this `Contains` is called for every request, I prefer to have a `map` over a `set` for performance reasons.

Reply:
`set.Set` uses a map for lookups internally.
It has a mutex and is thread-safe, so there's a slight performance penalty for that, but seems like a nice way to simplify this code a bit.